recipe/meta.yaml

```diff
@@ -36,7 +36,6 @@ requirements:
     - openfe-analysis ~=0.3.1
     - openff-toolkit >=0.16,<0.17
     - openmm !=8.1.0,>=8.0.0,<8.2
-    - openmmforcefields <0.15
```

Comment:
this one is actually fine, sorry I thought I linked to a different section in that slack message